### ISSUE TITLE
`--pushtag` option should push only the working tag, not all tags

### ISSUE THREAD

**Issue by bmcdonnell-ionx** [petervanderdoes/gitflow-avh#419]
Please make the `--pushtag` option to `git flow release finish` push only the tag for the release, _not_ all tags in the repo.

Pushing all tags is unexpected and undesirable behavior. It's unexpected because it is inconsistent with the context of the actions being performed, and also because the option name is singular (`--pushtag`, not `--pushtags`). It's undesirable because I maintain local tags which I have no intent to ever push. (e.g. `temp`, `experiment`, etc.)

Thanks.

**Comment by andidev** [comment]
yeah this is really causing a mess for us. all developers local tags being pushed and its gets really hard to remove a tag since if removed in origin it gets push again from another user next time they release.
